tests/libtest/lib1502.c

```diff
@@ -121,35 +122,36 @@ CURLcode test(char *URL)
 
 test_cleanup:
 
-#ifdef LIB1502
-  /* undocumented cleanup sequence - type UA */
-  curl_multi_cleanup(multi);
-  curl_easy_cleanup(easy);
-  curl_global_cleanup();
-#endif
-
-#ifdef LIB1503
-  /* proper cleanup sequence - type PA */
-  curl_multi_remove_handle(multi, easy);
-  curl_multi_cleanup(multi);
-  curl_easy_cleanup(easy);
-  curl_global_cleanup();
-#endif
-
-#ifdef LIB1504
-  /* undocumented cleanup sequence - type UB */
-  curl_easy_cleanup(easy);
-  curl_multi_cleanup(multi);
-  curl_global_cleanup();
-#endif
-
-#ifdef LIB1505
-  /* proper cleanup sequence - type PB */
-  curl_multi_remove_handle(multi, easy);
-  curl_easy_cleanup(easy);
-  curl_multi_cleanup(multi);
-  curl_global_cleanup();
-#endif
+  arg4 = libtest_arg4 ? atoi(libtest_arg4) : 0;
+  switch(arg4) {
+  case 0:
+  default:
+    /* undocumented cleanup sequence - type UA */
+    curl_multi_cleanup(multi);
+    curl_easy_cleanup(easy);
+    curl_global_cleanup();
+    break;
+  case 1:
+    /* proper cleanup sequence - type PA */
+    curl_multi_remove_handle(multi, easy);
+    curl_multi_cleanup(multi);
+    curl_easy_cleanup(easy);
+    curl_global_cleanup();
+    break;
+  case 2:
+    /* undocumented cleanup sequence - type UB */
+    curl_easy_cleanup(easy);
+    curl_multi_cleanup(multi);
+    curl_global_cleanup();
+    break;
+  case 3:
```

Comment:
Would it improve readability, if this number would be the test number? E.g. 3 → 1505.